formulaire/src/main/resources/public/ts/directives/question/question-type/question-type-multipleanswer/question-type-multipleanswer.html

```diff
@@ -0,0 +1,46 @@
+<div class="twelve">
+  <div class="choice" ng-repeat="choice in vm.question.choices.all | orderBy:['position', 'id']">
+    <div class="container-arrow" ng-if="vm.question.selected">
+      <div ng-class="{hidden : $first || choice.is_custom}" ng-click="vm.question.moveChoice(choice, vm.direction.UP)">
+        <i class="i-chevron-up lg-icon"></i>
+      </div>
+      <div ng-class="{hidden : $last || vm.question.choices.all[$index+1].is_custom}"
+           ng-click="vm.question.moveChoice(choice, vm.direction.DOWN)">
+        <i class="i-chevron-down lg-icon"></i>
+      </div>
+    </div>
+    <label class="twelve left-spacing-twice">
+      <input type="checkbox" id="check-[[choice.id]]" disabled>
+      <span style="cursor: default"></span>
+      <input type="text" class="width95 ten-mobile" ng-model="choice.value" ng-if="!vm.question.selected && !choice.is_custom"
+             placeholder="[[vm.i18n.getWithParam('formulaire.choice', choice.position)]]" disabled>
+      <input type="text" class="width95 ten-mobile" ng-model="choice.value" ng-if="vm.question.selected && !choice.is_custom"
+             placeholder="[[vm.i18n.getWithParam('formulaire.choice', choice.position)]]" input-guard>
+      <input type="text" class="width95 ten-mobile" ng-if="choice.is_custom"
+             value="[[vm.i18n.translate('formulaire.other')]] : " disabled>
+      <div class="image-container">
+<!--        <div class="image-wrapper">-->
+          <i class="i-close lg-icon" ng-click="vm.deleteImageSelect($index)" ng-if="vm.question.selected && !vm.hasFormResponses && choice.image !== ''"></i>
+          <image-select id="img{{$index}}" class="initialFloat cell"
+                        ng-if="vm.selectedChoiceIndex === $index && !vm.hasFormResponses"
+                        default="/img/illustrations/image-default.svg" ng-model="choice.image">
+          </image-select>
+          <img data-ng-src="[[choice.image]]"
```

Comment:
Tu dois pouvoir faire ça pour gérer le cas où l'image serait cassée :
```suggestion
          <img data-ng-src="[[choice.image || '/formulaire/public/img/empty-image.png']]"
```